adafruit_ads1x15/analog_in.py

```diff
@@ -64,3 +64,8 @@ def voltage(self) -> float:
         """Returns the voltage from the ADC pin as a floating point value."""
         volts = self.value * _ADS1X15_PGA_RANGE[self._ads.gain] / 32767
         return volts
+
+    def ADC_value(self, volts: float) -> int:
```

Comment:
```suggestion
    def convert_to_value(self, volts: float) -> int:
```

Maybe add `convert_to_volts` as well and use it in `.voltage`.